app/scripts/filters/eventAddressFormat.js

```diff
@@ -0,0 +1,39 @@
+import { allCountries } from 'country-region-data';
+
+angular
+  .module('confRegistrationWebApp')
+  .filter('eventAddressFormat', () => (city, state, zip, country) => {
+    const formattedState =
+      state &&
+      allCountries.reduce((result, currentCountry) => {
+        if (currentCountry[1] === country) {
+          result = currentCountry[2].filter(r => r[1] === state)[0][0];
+        }
+        return result;
+      }, '');
```

Comment:
Similar instance. Reduce doesn't seem appropriate here since you're searching for a single item and then operating on the single item you found.
```suggestion
      allCountries.find(currentCountry => currentCountry[1] === country)?.[2].filter(r => r[1] === state)?.[0][0];
```